lib/grp.gi

```diff
@@ -4620,6 +4629,23 @@ InstallMethod( MaximalNormalSubgroups, "for simple groups",
               function(G) return [ TrivialSubgroup(G) ]; end);
 
 
+#############################################################################
+##
+#M  MaximalNormalSubgroups( <G> )
+##
+InstallMethod( MaximalNormalSubgroups, "general method selection",
+              [ IsGroup ],
+    function(G)
+
+    if 0 in AbelianInvariants(G) then
+      # (p) is a maximal normal subgroup in Z for every prime p
+      Error("number of maximal normal subgroups is infinity");
```

Comment:
The .tst file basically should contain the first line of the error message.. but as @ChrisJefferson says, you can simply run the test file manually, and if you got it wrong, it'll tell you.

Just in case you don't know, you can run an individual test file with the `Test` functio (that's much quicker than running the whole `make testinstall` etc.)